scripts/nrf_profiler/calc_stats.py

```diff
@@ -7,35 +7,40 @@
 
 import argparse
 import logging
+import json
 
 
 def main():
-    parser = argparse.ArgumentParser(description='Calculating stats for events.',
+    parser = argparse.ArgumentParser(description="nRF Profiler event propagation statistics",
                                      allow_abbrev=False)
-    parser.add_argument('dataset_name', help='Name of dataset')
-    parser.add_argument('--start_time', help='Measurement start time[s]')
-    parser.add_argument('--end_time', help='Measurement end time[s]')
-    parser.add_argument('--log', help='Log level')
+    parser.add_argument("dataset_name", help="Name of nRF Profiler dataset")
+    parser.add_argument("test_presets", help="Test preset [*.json file]")
```

Comment:
breaking change, required argument.

You should  ensure that all scripts or documentation using this script are updated accordingly